packages/curve-ui-kit/src/themes/design/2_theme.ts

```diff
@@ -3,7 +3,7 @@ import { SurfacesAndText } from './1_surfaces_text'
 
 const { plain, inverted } = SurfacesAndText
 
-const Transition = `color ${TransitionFunction}, background-color ${TransitionFunction}, filter ${TransitionFunction}`
+const Transition = `color ${TransitionFunction}, background-color ${TransitionFunction}, filter ${TransitionFunction}, border ${TransitionFunction}`
```

Comment:
at what point is this going to be a transition for `all`? 🫠

Reply:
Also fine :laughing: